Three small changes: play/pause tooltip, mat icon typo, aria recommendation

joanise opened the pull request.

joanise committed:
fix: typo, refer to mat icon error_outline by its correct name

Using error_outline_outlined worked anyway, but the extra _outlined was
basically ignored and cause extra spaces to get inserted.

joanise committed:
fix: aria improvement suggested by Lighthouse Accessbility audit

joanise committed:
feat: tooltip to clarify that play and pause are the same button

joanise commented:
Note: I'm not sure my Spanish translation is OK, with "Déle play/pause", but I was adapting from the existing text. I'm hoping for a native speaker review before merging.

dhdaines reviewed (approved):
I will defer to a native speaker for the Spanish translation (which I find a bit shocking! where's the equivalent of the OQLF?) but aside from that this seems quite sensible.

joanise committed:
fix: fix spanish translation for play/pause

with native speaker review on Reproducir/pausar

and make the tooltips that are next to each other consistent
